back/node_watcher/src/tasks/createHeartbeat.ts

```diff
@@ -19,10 +19,10 @@ const createHeartBeat: Task<NomidotHeartBeat[]> = {
   name: 'createHeartBeat',
   read: async (
```

Comment:
@amaurymartiny how bad is it that now these arguments will be left unused in a lot of these tasks?

Reply:
Performace wise: there would be no difference. These args are passed by reference (aka they are not cloned), so there's close-to-0 memory overhead.

Reply:
Linting wise: yeah, we could add a rule that says if the variable is preceded by `_`, then it can be unused.